src/promptflow/tests/sdk_cli_azure_test/unittests/test_azure_cli_time_consume.py

```diff
@@ -1,15 +1,19 @@
 import multiprocessing
+import os
 import sys
 import timeit
 from unittest import mock
-
 import pytest
+from promptflow._core.operation_context import OperationContext
+from promptflow._constants import USER_AGENT
+from tests._constants import CI_CLI_USER_AGENT
 
 FLOWS_DIR = "./tests/test_configs/flows"
 DATAS_DIR = "./tests/test_configs/datas"
 
 
 def run_cli_command(cmd, time_limit=3600):
+    os.environ[USER_AGENT] = CI_CLI_USER_AGENT
```

Comment:
Since all our test run concurrently, maybe we need to run this in separate CI to avoid this change include other tests.

Reply:
Yes, I will modify the next PR and I need to consider deleting this agent without affecting the use of this agent by other ci  tests.